var/spack/repos/builtin/packages/hybpiper/package.py

```diff
@@ -17,7 +17,9 @@ class Hybpiper(Package):
 
     homepage = "https://github.com/mossmatters/HybPiper"
     url      = "https://github.com/mossmatters/HybPiper/archive/v1.2.0.tar.gz"
+    git      = "https://github.com/mossmatters/HybPiper/HybPiper.git"
```

Comment:
this isn't used, is it?

Reply:
Nope, but it also doesn't hurt. Makes it easier for someone to add develop if they need it.